src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue20250.cs

```diff
@@ -0,0 +1,25 @@
+﻿#if !WINDOWS // It seems the ClearPlaceholderIcon is not supported on Windows
```

Comment:
Could you create a bug? Taking a quick look into the code can see the implementation on Android and iOS but not on Windows.

Reply:
@jsuarezruiz, I created a new bug report for this and attached it here.